pkg/wwan/mmagent/vendor/modules.txt

```diff
@@ -322,7 +322,7 @@ github.com/lf-edge/eve-api/go/profile
 # github.com/lf-edge/eve/pkg/kube/cnirpc v0.0.0-20240315102754-0f6d1f182e0d
 ## explicit; go 1.20
 github.com/lf-edge/eve/pkg/kube/cnirpc
-# github.com/lf-edge/eve/pkg/pillar v0.0.0-20250504151246-ec39e287817b => ../../pillar
```

Comment:
Does this imply that we have a redirect in the go.mod file? Such redirects cause issues when generating the SBoM.

Reply:
Yes, this is only temporary and will be removed once this PR is merged. The alternative would be to split the pillar changes into a separate PR, which would need to be merged first.

Reply:
Ok